src/components/SearchBar.tsx

```diff
@@ -1,11 +1,13 @@
 import React, {Component} from 'react';
+import {NativeSyntheticEvent, TextInputChangeEventData} from 'react-native';
 import {View, TextInput, Image} from 'react-native';
 import SearchBarstyles from './SearchBarstyles';
 
 interface SearchBarProps {
   style?: object;
   placeholder: string;
   onChangeText: (text: string) => void;
+  onChange?: (e: NativeSyntheticEvent<TextInputChangeEventData>) => void;
```

Comment:
```suggestion
  onChangeText?: (value: string) => void;
```

зачем наружу выбрасывать ненужное, наружу только серч текст надо

Reply:
Зачем два метода, которые делают одно и то же?